.evergreen/config.yml

```diff
@@ -1919,20 +1919,20 @@ axes:
           - windows-64-vs2017-test
         variables:
           GCC_PATH: "/cygdrive/c/ProgramData/chocolatey/lib/mingw/tools/install/mingw64/bin"
-          GO_DIST: "C:\\golang\\go1.16"
+          GO_DIST: "C:\\golang\\go1.17"
           PYTHON3_BINARY: "C:/python/Python38/python.exe"
           VENV_BIN_DIR: "Scripts"
       - id: "ubuntu1604-64-go-1-16"
```

Comment:
For every value that you're changing to use a `GO_DIST` of `go1.17`, I think you'll want to change the `id` field to end with `go-1-17`.

Reply:
Good catch, will update.